### ISSUE TITLE
Provide text in strings.xml instead of images

### ISSUE THREAD

**Issue by HenriDellal** [alaskalinuxuser/app_critical_velocity#4]
It will help to translate the app and to experiment with text appearance. I can provide files for English and Russian languages.

**Comment by alaskalinuxuser** [comment]
That would be great! I am a little puzzled on how to do strings with libGDX, since it does not use the strings.xml file.

**Comment by alaskalinuxuser** [comment]
I will have to test this after I get back. Probably sometime next week. Looks great, but I am still going to have to learn up a bit on how to put it into the game...

Thanks!

**Comment by HenriDellal** [comment]
I've found some mistakes in the code written by me so I have to fix it. The good news is that I've added support of string arrays and finished file with English text.

**Comment by alaskalinuxuser** [comment]
Okay, I had to change everything for the text. Per:
https://github.com/libgdx/libgdx/wiki/Internationalization-and-Localization#creating-properties-files

The proper way to do internationalizaion (i18n)/localization (i10n) is with bundles. So, I made this commit:
https://github.com/alaskalinuxuser/app_critical_velocity/commit/3a490344b91c5019594a92f46a5548061692efd7

To add more languages, make a new MyStrings_<language>_<locale>.properties file. Example is the MyStrings_en_us.properties file in the assets folder.

I will make one more commit to remove the call to set the local to "en_us" which I was using for testing.
I don't like the way it looks, but it works. Maybe someone could clean it up a bit, or I could do that sometime in the future.

Hope that helps!
